src/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard/Properties/PublishProfiles/Web Deploy to Azure.pubxml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+This file is used by the publish/package process of your Web project. You can customize the behavior of this process
+by editing this MSBuild file. In order to learn more about this please visit https://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <WebPublishMethod>MSDeploy</WebPublishMethod>
```

Comment:
what is this for?

Reply:
Let's you deploy straight to Azure from the UI. This file is mostly autogenerated.